spec/routes/web/project/postgres_spec.rb

```diff
@@ -919,7 +919,7 @@
         click_button "Save"
 
         expect(page).to have_flash_error "Validation failed for following fields: pg_config.work_mem"
-        expect(page).to have_content "must match pattern: ^[0-9]+(kB|MB|GB|TB)?$"
+        expect(page).to have_content "must match pattern: \\A[0-9]+(kB|MB|GB|TB)?\\z"
```

Comment:
Questionable whether we should expose ruby regexp specific metacharacters in user error messages. I think it would be best to remove the `\A` and `\z` when displaying the pattern.